src/test/scala/com/microsoft/hyperspace/goldstandard/TPCDSBase.scala

```diff
@@ -0,0 +1,575 @@
+// scalastyle:off
+
+/**
+ * This trait was built upon: https://github.com/apache/spark/blob/v3.1.1/sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala.
+ *
+ * The below license was copied from: https://github.com/FasterXML/jackson-module-scala/blob/2.10/src/main/resources/META-INF/LICENSE
+ */
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.goldstandard
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.catalyst.TableIdentifier
+import org.apache.spark.sql.internal.SQLConf
+
+import com.microsoft.hyperspace.SparkInvolvedSuite
+
+trait TPCDSBase extends SparkFunSuite with SparkInvolvedSuite {
+
+  val conf = SQLConf.get
+
+  // The TPCDS queries below are based on v1.4.
+  // TODO: Fix bulid pipeline for q49 and reenable q49.
+  val tpcdsQueries = Seq("q1")
```

Comment:
Note: this is a complete list of queries which will run as part of this test. Currently only q1 is selected. In subsequent prs, all queries will be enabled